models/project/src/mc-project/applied-permission/generator.ts

```diff
@@ -0,0 +1,24 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import type { TAppliedPermission } from './types';
+
+const fakePermissions = [
+  'canViewProducts',
+  'canManageProductTypes',
+  'canViewStates',
+  'canManageStates',
+  'canViewOrders',
+  'canManageOrders',
+  'canAddOrders',
+  'canAddDiscountCodes',
+  'canManageProjectSettings',
+];
+
+const generator = Generator<TAppliedPermission>({
+  fields: {
+    // TODO: use more realistic values for action rights
```

Comment:
It seems we don't need this comment anymore.

Reply:
updated [d22c170](https://github.com/commercetools/test-data/pull/464/commits/d22c1709dcdb112fb8b998edf20afcaf2328be63)